static/app/app/controller.js

```diff
@@ -14,13 +14,13 @@ var NameView = require('app/edit/views').NameView;
 
 var Controller = Marionette.Object.extend({
     initialize: function(){
-        channel.on('call:method', function(methodName){
+        channel.on('call:method', function(methodName, saveAlert){
```

Comment:
The `call:method` trigger gets used with all of the controller methods. Instead of specifically referring to the second parameter as `saveAlert` it would be better to use a generic name like `arg`. Even better would be to not name them at all and use the `arguments` object and `apply` to dynamically assign the arguments.